site/i18n.zh-Hant.js

```
@@ -0,0 +1,1042 @@
+  'Catalog': '課程目錄',
+  'Roadmap': '學習藍圖',
+  'Glossary': '詞彙表',
+  'About': '關於',
+  'Books': '書籍',
+  'Home': '首頁',
+  'Course': '課程',
+  'Lesson': '單元',
+  'Lessons': '單元',
+  'Phase': '階段',
+  'Type': '類型',
+  'Language': '語言',
```

Biome (2.5.5)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)